src/main/java/org/sagebionetworks/EvaluationUtils.java

```diff
@@ -108,15 +112,26 @@ public static SubmissionStatusEnum getFinalSubmissionState() {
 
 	public static String getStringAnnotation(SubmissionStatus status, String key) {
 		Annotations annotations = status.getAnnotations();
-		if (annotations==null) return null;
+		if (annotations == null) return null;
 		List<StringAnnotation> sas = annotations.getStringAnnos();
-		if (sas==null) return null;
+		if (sas == null) return null;
 		for (StringAnnotation sa : sas) {
 			if (sa.getKey().equals(key)) return sa.getValue();
 		}
 		return null;
 	}
 
+	public static String getStringAnnotationV2(SubmissionStatus status, String key) {
```

Comment:
We probably do not need the getters at this stage, for now we just need to make sure that we save the annotations V2 together with the old annotations V1, but the underlying engine that still uses the old annotations must be kept in place to be backward compatible.